web/components/modals/ChildModal.tsx

```diff
@@ -87,19 +120,24 @@ function ChildModal({
                     <p>Sex</p>
                     <select
                       className="w-full bg-[#FAFBFC] border-[#D9D9D9] border-2 rounded py-2 px-2 focus:outline-0 min-h-[40px]"
+                      defaultValue={values?.sex || "select"}
                       {...register("sex", { required: true })}
                     >
                       <option value="select">Select</option>
                       <option value="female">Female</option>
                       <option value="male">Male</option>
                     </select>
                   </div>
+                  <div className="form-group mb-3 w-full col-span-2 ">
+                    <SearchableDropdown options={caregivers}
+                    label="Caregiver" placeholder={defaultCaretaker?.label ?? "Select"} onChange={handleCaretakerChange}/>
```

Comment:
The SearchableDropdown looks great and works really well! This is such a small thing but can you change the placeholder text to "Search for Caregiver" to make it more obvious that users need to Search?